maven-plugin/plugin-core/src/test/java/org/jvnet/jaxb/maven/util/IOUtilsTests.java

```diff
@@ -106,7 +107,7 @@ public void reorderFilesIncludesWithWildcardMiddle() {
 
     @Test
     public void reorderFilesIncludesWithWildcardLast() {
-        String[] files = {"a.xsd", "common/c1.xsd", "b.xsd", "common/c2.xsd", "common/a.xsd", "common/b.xsd" };
+        String[] files = {"a.xsd", "common" + File.separatorChar + "c1.xsd", "b.xsd", "common" + File.separatorChar + "c2.xsd", "common" + File.separatorChar + "a.xsd", "common" + File.separatorChar + "b.xsd" };
         List<String> orderedFiles = IOUtils.reorderFiles(files, new String[] { "a.xsd", "b.xsd", "common/*.xsd" });
```

Comment:
Does this "/" also need to be replaced?

Reply:
No, it's treated in `reorderFiles` function with special `normalizePath` function but the input array should be platform dependant

Reply:
Waiting for @vianneyd to confirm resolution

Reply:
I confirm everythings now pass on my laptop ;) 
Thanks!